### DISCUSSION TITLE
How to Restrict user from editing preference?

### DISCUSSION THREAD

**Discussion by shawkatmarrouf** [YetiForceCompany/YetiForceCRM#16775] [How to] [answered]
I want to restrict specific users from editing their preference, is there a way?

**Comment by rskrzypczak** [comment, marked as answer]
Edit config file: 
https://github.com/YetiForceCompany/YetiForceCRM/blob/c1af51b085dd3b48312ab15945f1e52ea58165d4/config/Security.php#L29-L30
